compiler/hash-lower/src/build/matches/declarations.rs

```diff
@@ -106,7 +106,7 @@ impl<'tcx> BodyBuilder<'tcx> {
             }
             Pat::Ctor(CtorPat { ctor_pat_args: fields, ctor_pat_args_spread: spread, .. })
             | Pat::Tuple(TuplePat { data: fields, data_spread: spread }) => {
-                fields.borrow().iter().for_each(|field| {
+                fields.borrow().borrow().iter().for_each(|field| {
```

Comment:
Whats the second `borrow()` for? What does `fields.borrow()` return?

Reply:
`fields: ParamsId`, so `fields.borrow(): &Node<ParamsSeqId>`, so `fields.borrow().borrow(): &[Node<ParamId>]`. Maybe I will change this to `.members().borrow()` since that does the same thing, but is more clear.

Reply:
Yeah this already exists, i just need to change these .borrow().borrow() call sites.